src/utils/utils.ts

```diff
@@ -114,3 +114,37 @@ export const calculateDuration = (startTime: Date | string, endTime: Date | stri
 
   return `${hoursString}${hoursString && minutesString ? ' ' : ''}${minutesString}${secondsString}`;
 };
+
+// Mathematical base to work in. Currently: Binary units (i.e. KiB etc). To use KB / MB etc switch to decimal (10).
```

Comment:
According to [JEDEC definition of kilo](https://www.jedec.org/standards-documents/dictionary/terms/kilo-k-prefix-units-semiconductor-storage-capacity) and  [JEDEC definition of mega](https://www.jedec.org/standards-documents/dictionary/terms/mega-m-prefix-units-semiconductor-storage-capacity), `K` refers to 2^10 instead of 10^3, you should use `k` instead. while `M` is equal to 10^6 and `G` is equal to 10^9. So you can change it to:
`// Mathematical base to work in. Currently: Binary units (i.e. KiB etc). To use kB / MB etc switch to decimal (10).`